src/views/OrderDetail.vue

```diff
@@ -0,0 +1,225 @@
+              <ion-label></ion-label>
+            </ion-chip>
+            <ion-chip >
+              <ion-icon />
+              <ion-label></ion-label>
+            </ion-chip>
+            <ion-checkbox @click="checkGroupedProducts(id)" />
+            <ion-button fill="clear" color="medium">
+              <ion-icon  :icon="ellipsisVerticalOutline" />
+            </ion-button>
+          </div>
+          <div class="list-item">
+            <ion-item  lines="none">
+              <ion-thumbnail>
+                <Image :src="item.imageUrl" />
+              </ion-thumbnail>
+              <ion-label>
+                {{ item.internalName }}
+              </ion-label>
+            </ion-item>
+            <ion-chip outline>
+              <ion-label>{{ item.isNewProduct? $t("Preorder") : $t("Backorder") }}</ion-label>
+            </ion-chip>
+            <ion-chip outline>
+              <ion-label>{{ item.quantityOrdered }} {{ $t("Ordered") }}</ion-label>
+            </ion-chip>
+            <ion-chip outline>
+              <ion-icon :icon="sendOutline" />
+              <ion-label>{{ item.arrivalDate }}</ion-label>
+            </ion-chip>
+              <ion-checkbox v-if="listOfCheckedProducts.includes(item.shopifyproductSKU)" checked="true" @click="onChange(item.shopifyproductSKU)"/>
+            <ion-checkbox v-else @click="onChange(item.shopifyproductSKU)"/>
+            <ion-button fill="clear" color="medium">
+              <ion-icon slot="icon-only" :icon="ellipsisVerticalOutline" />
+            </ion-button>
+          </div>
+        </div>
+      </div>
+    </ion-content>   
+  </ion-page>
+</template>   
+<script lang="ts">
+import Image from '@/components/Image.vue';
+
+import { defineComponent } from 'vue';
+import { mapGetters, useStore } from "vuex";
+import { useRouter } from 'vue-router';
+import { DateTime } from 'luxon';
+import { IonPage, IonHeader, IonToolbar, IonBackButton, IonTitle, IonContent, IonSearchbar, IonItem, IonThumbnail, IonLabel, IonInput, IonChip, IonIcon, IonButton, IonCheckbox, IonSelect, IonSelectOption } from '@ionic/vue'
+import { ellipsisVerticalOutline, sendOutline, checkboxOutline, arrowUndoOutline } from 'ionicons/icons'
+export default defineComponent({
+    components: {
+      Image,
+      IonPage,
+      IonHeader,
+      IonToolbar,
+      IonBackButton,
+      IonTitle,
+      IonContent,
+      IonSearchbar,
+      IonItem,
+      IonThumbnail,
+      IonLabel,
+      IonInput,
+      IonChip,
+      IonIcon,
+      IonButton,
+      IonCheckbox,
+      IonSelect,
+      IonSelectOption
+    },
+    computed: {
+      ...mapGetters({
+        orderItems: 'order/getOrderItems',
+        getProduct: 'product/getProduct',
+      }),
+    },
+    data() {
+      return {
+        numberOfDays: 0,
+        numberOfpieces: 0,
+        catalog: "",
+        listOfCheckedProducts: [] as any,
+        originalCsv: {}
+      }
+    },
+    methods: {
+      onChange(productSku: string) {
```

Comment:
Improve method name. It should reflect the operation

Reply:
Improved method name.